docs/using_vfxt_py.md

```diff
@@ -1,237 +1,270 @@
 
 ### Additional Create Settings
 
-This section describes additional options that can be useful when creating a new vFXT cluster.
+This section describes additional options that can be useful when creating a new vFXT cluster. 
 
 **Skip Cleanup**
 
 If there is an error, vfxt.py rolls back what was done. In some situations, you might want to prevent this rollback - for example, during troubleshooting. The `--skip-cleanup` option leaves nodes, buckets, routes, roles, and other entities in the state they had when the error occurred. Anything created during the operation is not removed.
 
-    vfxt.py --cloud-type azure    \
-      <authentication options>   \
-      <environment options>      \
-      <action>                   \
-      --skip-cleanup           
+```bash
+
+vfxt.py	--cloud-type type         \
+	<authentication options>  \
+	<environment options>     \
+	<action>                  \
+	--skip-cleanup           
+```
 
 **No Core Filer**
 
-vfxt.py can skip the creation of a bucket and the associated cloud core filer configuration. Use this option if you want to create a vFXT cluster and configure its storage afterward by using the Avere Control Panel or the XML-RPC command-line cluster configuration API.
+vfxt.py can skip the creation of a bucket and the associated cloud core filer configuration. Use this option if you want to create a vFXT cluster and configure its storage afterward by using the Avere Control Panel or the XML-RPC command-line cluster configuration API. 
+
+> Note: If you use the `--nfs-mount` option, vfxt.py does not attempt to create a cloud core filer. 
 
-**NOTE:** If you use the `--nfs-mount` option, vfxt.py does not attempt to create a cloud core filer.
+```bash
 
-    vfxt.py --cloud-type azure     \
-      <authentication options>    \
-      <environment options>       \
-      <action>                    \
-      --no-corefiler
+vfxt.py	--cloud-type type           \
+	<authentication options>    \
+	<environment options>       \
+	<action>                    \
+	--no-corefiler
+```
 
 **NFS Core Filer**
 
-vfxt.py can configure an NFS core filer at cluster creation time if you provide the NFS mount point in the host:/path format. Note that if you specify an NFS core filer, vfxt.py does not create a cloud core filer.  If your storage appliance type is one of the values in `--nfs-type` you can use that option here to set it. (If you don’t set the `--nfs-type` option it defaults to "other".)
+vfxt.py can configure an NFS core filer at cluster creation time by providing the NFS mount point in the host:/path format. Note that if you specify an NFS core filer, vfxt.py does not create a cloud core filer.  If your storage appliance type is one of the values in `--nfs-type` you can use that option here to set it. (If you don’t set the `--nfs-type` option it defaults to other.)  
```

Comment:
First sentence my grammar was more correct.
Last sentence shouldn't 'other' be formatted as a literal value?